src/components/form/fields/phoneNumberField.tsx

```diff
@@ -182,7 +179,8 @@ const phoneNumberField = (
                 }
             },
             unbind: formValue => {
-                return (typeof formValue === 'object' && 'raw' in formValue ? formValue.raw : formValue)
+                const value = (typeof formValue === 'object' && 'raw' in formValue ? formValue.raw : formValue)
+                return value == "" ? undefined : value
             }
```

Comment:
`format.bind/unbind` are value transformers between field data model and raw data passed to from (submit to form handler).